src/used.md

```diff
@@ -0,0 +1,62 @@
+## #[used]
```

Comment:
There's no reason for this to have its own page. Or anything listed in the ABI section. I'd suggest merging the two pages.